checks.d/network.py

```diff
@@ -176,6 +176,8 @@ def _check_linux(self, instance):
                 metrics = self._parse_linux_cx_state(lines[2:], self.TCP_STATES['netstat'], 5)
                 for metric, value in metrics.iteritems():
                     self.gauge(metric, value)
+            except Exception:
+                self.log.exception("Error collecting connection stats.")
```

Comment:
We should print the exception here as well, so we don't silence other important Exceptions